### PR TITLE
I did something

### PR DESCRIPTION
**Pull request by kndshein**

### CODE REVIEW

File: src/pages/Game.module.scss [hunk 1]
```diff
@@ -64,15 +64,16 @@
 .prev_mobile,
 .next_mobile {
   display: none;
-  width: 4rem;
+  width: 3rem;
+  min-width: 3rem;
```
Comment (daleinen7) [inline src/pages/Game.module.scss:68]: Isn't one of these redundant?

File: src/styles/_variables.scss [hunk 2]
```diff
@@ -24,20 +24,25 @@ $dark-grey: rgb(102, 114, 143);
   border-radius: 10px;
   box-shadow: 3px 3px darken($color, 25%);
 
+  &.disabled {
+    pointer-events: none;
+    color: $dark-grey;
+    background-color: $grey;
+    box-shadow: none;
```
Comment (daleinen7) [inline src/styles/_variables.scss:31]: could add `cursor: not-allowed;`

**Review by daleinen7 — APPROVED** [review]
I'm just being real nitpicky now. I'm just going to improve this.

File: src/styles/_variables.scss (hunk 2 above)
Comment (kndshein) [reply src/styles/_variables.scss:31]: I've never seen that before. I just tried it, but `cursor: not-allowed` still allows users to hover over the buttons and the cursor changes to "Do not disturb" icon.

File: src/pages/Game.module.scss (hunk 1 above)
Comment (kndshein) [reply src/pages/Game.module.scss:68]: You're absolutely right. Fixed.

**Pull request merged**